tests/integration/update_cluster/privatecalico/cloudformation.json.extracted.yaml

```diff
@@ -0,0 +1,490 @@
+Resources.AWSAutoScalingLaunchConfigurationmasterustest1amastersprivatecalicoexamplecom.Properties.UserData: |
```

Comment:
@rifelpet just for my own understanding, is this copied from elsewhere or generated somehow?

Reply:
It is generated. I think some of our tests have an env var you can set that will write the actual output to the expected files so you can make updates like this easily, but I don’t think these integration tests support that. I ended up just making these files empty, ran the tests, then copied the diff outputted by the failing test into the files.